SeleniumBasicsTest/appSource/app_tecklist.py

```diff
@@ -0,0 +1,72 @@
+from selenium.webdriver.support.select import Select
+
+from libCommon.selenium_helper import SeleniumHelper
+import time
+from sourcelocators.techlocators import Techlistlocators
+
+class APPTechList:
+
+    def __init__(self):
+        self.url = "https://www.techlistic.com/p/selenium-practice-form.html"
+        self.driver = SeleniumHelper()

```

Comment:
already driver object initialize in seleniumbasic use it here and proper name gave.
ex: self.helper = seleniumHelper()
self.driver = self.helper.inistilize_driver
then use it here

Reply:
i will change as per the error and i will PR
thanks very much